boards/infineon/cyw920829m2evk_02/doc/index.rst

```diff
@@ -117,7 +142,101 @@ The path to the installed Infineon OpenOCD executable must be available to the `
             west flash
             west debug
 
-Once the gdb console starts after executing the west debug command, you may now set breakpoints and perform other standard GDB debugging on the CYW20829 CM33 core.
+Once the gdb console starts after executing the west debug command, you may now set breakpoints and
+perform other standard GDB debugging on the CYW20829 CM33 core.
+
+Operate in SECURE Lifecycle Stage
+*********************************
+
+The device lifecycle stage (LCS) is a key aspect of the security of the AIROC™
+CYW20829 Bluetooth® MCU. The lifecycle stages follow a strict, irreversible progression dictated by
+the programming of the eFuse bits (changing the value from "0" to "1"). This system is used to
+protect the device's data and code at the level required by the user.
+SECURE is the lifecycle stage of a secured device.
+Follow the instructions in `AN239590 Provision CYW20829 to SECURE LCS`_ to transition the device
+to SECURE LCS. In the SECURE LCS stage, the protection state is set to secure. A secured device
+will only boot if the authentication of its flash content is successful.
+
+The following configuration options can be used to build for a device which has been provisioned
+to SECURE LCS and configured to use an encrypted flash interface:
+
+- ``CONFIG_INFINEON_SECURE_LCS=y``: Enable if the target device is in SECURE LCS
```

Comment:
use
```
:kconfig:option:`CONFIG_blah...`
```
and just say to set them not with the =blah part, so that they have links to the actual Kconfig descriptions